src/webui/api/torrentscontroller.cpp

```diff
@@ -248,6 +249,34 @@ namespace
         return {dht, pex, lsd};
     }
 
+    QJsonArray getAllTrackers(const BitTorrent::Torrent *const torrent, const bool includeSticky)
```

Comment:
Let's not handle "sticky trackers" in it:
```suggestion
    QJsonArray getTrackers(const BitTorrent::Torrent *const torrent)
```

Reply:
Also move the function below `getStickyTrackers`, i.e. to line 206.

Reply:
Ok so keep the "fake" trackers in there. 

> Also move the function below `getStickyTrackers`, i.e. to line 206.

Below line 206 would mean to put the logic inside the getStickyTrackers function?

Reply:
>Ok so keep the "fake" trackers in there.

Sorry, I wasn't clear. I meant don't add the "sticky trackers" in `getTrackers()`. Only let `getStickyTrackers()` return it.

>Below line 206 would mean to put the logic inside the getStickyTrackers function?

No. Like this:
```c++
QJsonArray getStickyTrackers(const BitTorrent::Torrent *const torrent)
{
    // ...
}

QJsonArray getTrackers(const BitTorrent::Torrent *const torrent)
{
    // ...
}
```

Reply:
Ah! That's more clear indeed!

Updated that, and added a merge in `trackersAction` to keep the current way data is returned in the api, like this:

![image](https://github.com/user-attachments/assets/f93a696d-4320-45b4-a42f-ec4ff8c068f9)

Not sure if this is the ideal way to do it or not https://github.com/qbittorrent/qBittorrent/pull/22128/files#diff-2c6fdae7a7c5a881f0a502368050ed2f7c7c9173e479a2b0921cf0ca8740a8efR572-R576